options.go

```diff
@@ -0,0 +1,267 @@
+package envbuilder
+
+import (
+	"github.com/coder/serpent"
+)
+
+// Options contains the configuration for the envbuilder.
+type Options struct {
+	SetupScript          string
```

Comment:
I think this change is quite unfortunate as it removes all documentation from Go docs (comment above field).

It'd be nice if we could keep the comments on the struct and instead `go generate` the serpent descriptions from those.

If we don't amend this (comments on fields), it doesn't really make sense to keep this a public/exported struct.

Reply:
> It'd be nice if we could keep the comments on the struct and instead go generate the serpent descriptions from those.

I'm happy to see `go generate` adoption here, but maybe in a separate issue/PR? Let's not overwhelm Bruno with extra magical Go spells.

Reply:
Marking this as resolved for now. Going to comment what are all the follow-ups later.